modules/core/src/utils/json-loader.js

```diff
@@ -9,6 +9,7 @@ export default {
   name: 'JSON',
   // TODO - can we stream process geojson?
   extensions: ['json', 'geojson'],
+  mimeTypes: ['application/json'],
```

Comment:
GeoJSON's [official `mimeType`](https://tools.ietf.org/html/rfc7946#section-12) is `application/geo+json`. Would probably be good to add given that we allow the `.geojson` extension.